metadata-service/factories/src/main/java/com/linkedin/metadata/boot/OnBootApplicationListener.java

```diff
@@ -62,7 +62,7 @@ public void onApplicationEvent(@Nonnull ContextRefreshedEvent event) {
   public Runnable isSchemaRegistryAPIServletReady() {
     return () -> {
         final HttpGet request = new HttpGet(provider.getKafka().getSchemaRegistry().getUrl());
-        int timeouts = 30;
+        int timeouts = 60;
```

Comment:
Can this get pushed into a configuration? 😄

Reply:
I think so.